index.php

```diff
@@ -0,0 +1,38 @@
+<?php
+
+
+$db = new PDO('mysql:host=db; dbname=ukgamesfunded', 'root', 'password');
+
+$db->setAttribute(PDO::ATTR_DEFAULT_FETCH_MODE, PDO::FETCH_ASSOC);
```

Comment:
I would suggest putting these two lines of code into your own re-usable function. The benefit of doing this is that you could re-use the function on any new pages that also require a database connection, rather than having to copy and paste it each time.

The function should run these two lines of code before returning $db.